tests/e2e_tests/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended"],
+  "rules": {
+    "prettier/prettier": 2,
+    "consistent-this": ["error", "self"],
+    "import/no-named-as-default": 0,
+    "no-console": 0,
+    "no-multiple-empty-lines": 2,
+    "no-prototype-builtins": 0,
+    "quotes": [2, "single", { "allowTemplateLiterals": true }],
+    "sonarjs/cognitive-complexity": 0,
```

Comment:
I think that we should have it. all the rest of the codebase has some sort of complexity restrictions. `["error", 8]` ?

Reply:
that rule unfortunately doesn't really work with react components and is more targeted at plain JS - so no.